src/lib.rs

```diff
@@ -72,45 +74,32 @@ impl State {
         let retry_enabled = self.config.retry_request_config;
         let retry_attempts = self.config.user_creation_retry_attempts;
 
-        let mut handles = FuturesUnordered::new();
-
         let progress_bar = create_progress_bar(
             "Init users".to_string(),
             (desired_users - actual_users).try_into().unwrap(),
         );
         progress_bar.tick();
 
-        for i in actual_users..desired_users {
-            handles.push(tokio::spawn({
-                let server = server.clone();
-                let metrics = self.metrics.clone();
-                let progress_bar = progress_bar.clone();
+        let range = (actual_users..desired_users).collect::<Vec<usize>>();
```

Comment:
Not sure the `collect` into a `Vec` is needed here, we can iter the range as is.

Reply:
Fixed!